### DISCUSSION TITLE
Problem with the IpGrabber

### DISCUSSION THREAD

**Discussion by Thibult** [kl3ssydra/kl3ssydra#4] [Q&A]
Hi !
I would like to use the IpGrabber but, when i run the command, it start a new command prompt which disappears instantly after.
Then, the page initial page says ''[#] Error''
I've installed everything right, every depencies (think so)
Could you help me please ?

**Comment by kl3ssydra** [comment]
it's a ngrok problem. when the grabber opens the ngrok pannel u have to remain on the main console
